src/components/ArticlesTable/ArticleTitle.js

```diff
@@ -0,0 +1,99 @@
+import { Component, PropTypes } from 'react';
+import TextField from 'material-ui/TextField';
+import { TableRowColumn } from 'material-ui/Table';
+
+import styles from './ArticlesTable.less';
+
+const inlineStyles = {
+  wrapWordColumn: {
+    whiteSpace: 'normal',
+    wordWrap: 'break-word',
+    width: '200px',
+    paddingLeft: '0',
+    paddingRight: '0',
+  },
+};
+
+export class ArticleTitle extends Component {
+  static propTypes = {
+    read: PropTypes.string,
+    title: PropTypes.string,
+    url: PropTypes.string,
+    onTitleChanged: PropTypes.func,
+  };
+
+  constructor() {
+    super();
```

Comment:
https://facebook.github.io/react/docs/state-and-lifecycle.html#adding-local-state-to-a-class
> Class components should always call the base constructor with props.